plugins/scroll-options/src/ScrollBlockDragger.js

```diff
@@ -96,53 +96,6 @@ export class ScrollBlockDragger extends Blockly.BlockDragger {
     this.scrollDirections_ = ['top', 'bottom', 'left', 'right'];
   }
 
-  /**
-   * Whether the behavior to scroll the workspace when a block is dragged near
-   * the edge is enabled.
-   * @type {boolean}
-   * @public
-   */
-  static edgeScrollEnabled = true;
-
-  /**
-   * Configuration options for the scroll-options settings.
-   * @type {!EdgeScrollOptions}
-   * @protected
-   */
-  static options_ = defaultOptions;
-
-  /**
-   * Update the scroll options. Only the properties actually included in the
-   * `options` parameter will be set. Any unspecified options will use the
-   * previously set value (where the initial value is from `defaultOptions`).
-   * Therefore, do not pass in any options with explicit `undefined` or `null`
-   * values. The plugin will break. Just leave them out of the object if you
-   * don't want to change the default value.
-   *
-   * This method is safe to call multiple times. Subsequent calls will add onto
-   * previous calls, not completely overwrite them. That is, if you call this
-   * with:
-   *
-   *     `updateOptions({fastMouseSpeed: 5});
-   *     updateOptions({slowMouseSpeed: 2});`.
-   *
-   * Then the final options used will include both `fastMouseSpeed: 5` and
-   * `slowMouseSpeed: 2` with all other options being the default values.
-   * @param {!EdgeScrollOptions} options Object containing any or all of
-   *     the available options. Any properties not present will use the existing
-   *     value.
-   */
-  static updateOptions(options) {
-    ScrollBlockDragger.options_ = {...ScrollBlockDragger.options_, ...options};
-  }
```

Comment:
Static _methods_ really ought to be supported by Babel, since they've been around since ES6 (2015).  Did you move this one (and `resetOptions`, below) out of the class body only because it references the `.options_` property and the linter was complaining about use-before-declaration?

Reply:
I didn't try moving them separately. Babel just quit when it found the first error (e.g. it only errored on the first property instead of giving me every error in the file) so I didn't try it to see if the static methods would have been fine because I didn't know there was a difference. 

I didn't see any use-before-declaration errors / lint messages, and I am using `options` before declaring it even now. We have a different eslint config for samples though and it may just not be turned on here.

Reply:
> I didn't see any use-before-declaration errors / lint messages

Great.  In core it was being (in my view) excessively zealous about that.